app/mailers/devise_mailer.rb

```diff
@@ -1,9 +1,9 @@
 class DeviseMailer < Devise::Mailer
-  before_action :use_site_config_values
+  before_action :use_settings_general_values
 
-  def use_site_config_values
+  def use_settings_general_values
     Devise.mailer_sender =
-      "#{SiteConfig.community_name} <#{SiteConfig.email_addresses[:default]}>"
-    ActionMailer::Base.default_url_options[:host] = SiteConfig.app_domain
+      "#{Settings::General.community_name} <#{Settings::General.email_addresses[:default]}>"
```

Comment:
Same here with the _community_name_. I could be _totally_ misunderstanding something though.

```suggestion
      "#{Settings::Community.community_name} <#{Settings::General.email_addresses[:default]}>"
```